components/Marker/Marker.tsx

```diff
@@ -15,47 +14,49 @@ const Marker = ({
   country,
   author,
   photo,
-  date
+  date,
+  orientation = 'default' // Default to 'default' orientation
```

Comment:
```suggestion
  orientation = 'vertical'
```
Vertical should be the default orientation